### PR TITLE
Improve OGM about page

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: ogm/index.html.haml [hunk 1]
```diff
@@ -8,7 +8,7 @@ project_hero_partial: project/ogm/hero-text.html.haml
 -# Center body
 .feature-block
   :asciidoc
-    == Many NoSQL stores, one standard way to access them
+    == Many NoSQL stores, one API to access them
```
Comment (gsmet) [inline ogm/index.html.haml:11]: This is just to avoid having the title on 2 lines.

File: ogm/index.html.haml [hunk 2]
```diff
@@ -17,20 +17,27 @@ project_hero_partial: project/ogm/hero-text.html.haml
 
     == Wide range of backends
 
-    OGM talks to NoSQL backends via store-specific dialects. Currently there is support for
+    OGM talks to NoSQL backends via store-specific dialects. In OGM proper, there is support for
```
Comment (DavideD) [inline ogm/index.html.haml:20]: I think I never heard the expression:`In OGM proper`. what do you mean by it?

Maybe we can just remove the `Currently,` and use `It supports ...`

**Comment by DavideD** [comment]
Looks good apart for the single comment

**Comment by gsmet** [comment]
Merged after a discussion with Davide.

**Pull request closed without merging**